src/Tickets/Flexible_Tickets/Custom_Tables.php

```diff
@@ -13,11 +13,7 @@
 use TEC\Common\StellarWP\DB\DB;
 use TEC\Common\StellarWP\Schema\Config as Schema_Config;
 use TEC\Common\StellarWP\Schema\Register as Schema_Register;
-use TEC\Tickets\Flexible_Tickets\Custom_Tables\Capacities;
-use TEC\Tickets\Flexible_Tickets\Custom_Tables\Capacities_Relationships;
-use TEC\Tickets\Flexible_Tickets\Custom_Tables\Posts_And_Posts;
 use TEC\Tickets\Flexible_Tickets\Custom_Tables\Posts_And_Ticket_Groups;
-use TEC\Tickets\Flexible_Tickets\Custom_Tables\Posts_And_Users;
```

Comment:
Do we have to add these back?

Reply:
No, these are classes that don't exist and aren't referenced in the file.